src/test/java/core/basesyntax/servicetests/MyReaderTest.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax.servicetests;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.service.reader.MyReaderImpl;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class MyReaderTest {
+    private static MyReaderImpl myReader;
+
+    @BeforeClass
+    public static void beforeClass() {
+        myReader = new MyReaderImpl();
+    }
+
+    @Test
+    public void fileReader_correctFile_ok() {
```

Comment:
во всех тестах консистентный нейминг: сначала пишем имя метода, который тестируем
```suggestion
    public void readFromFile_correctFile_ok() {
```